test_routes.py

```diff
@@ -0,0 +1,14 @@
+from flask import Blueprint
+from flask import jsonify
+from . import mongo
+
+test_routes = Blueprint('test_routes', __name__)
+
+
+@test_routes.route('/users', methods=['GET'])
```

Comment:
So the way the structure works is that each file will be a "Blueprint" of sorts for routes of specific types. For example, we have an `auth.py` (for authentication purposes) and a `test_routes.py` (for testing out the db and whatever else). I can teach you how this works on discord sometime @DarienFlamont

Reply:
Ok makes sense.  Will we need a separate python file for every GET and POST request?